src/pages/tutorials/index.tsx

```diff
@@ -1,68 +1,45 @@
-import pageContent from 'content/tutorials-landing.json'
+import pageData from 'content/tutorials-landing.json'
+import { stripUndefinedProperties } from 'lib/strip-undefined-props'
 import { getCollections } from 'lib/learn-client/api/collection'
 import TutorialsLandingView from 'views/tutorials-landing'
 
 const getStaticProps = async () => {
+	// Build up list of collection slugs to fetch
 	const collectionSlugsToFetch = new Set<string>()
-
-	Object.values(pageContent).forEach(({ featuredCollectionSlugs }) => {
+	Object.values(pageData).forEach(({ featuredCollectionSlugs }) => {
```

Comment:
I largely think the way we have structured our JSON & this `Object.values()` treatment is a bit of an anti-pattern.

It makes the `tutorials-landing.json` object subject to an really restrictive interface where every field within the top-level object must be of the same type.  In the event we wanted to move more data into that object (for example, the "Better Together" collections), we couldn't without breaking these references here.  I think it's reasonable to imagine authors would want control of that as well.

I think the right way to structure that data would to have an array of products, ex:

```json
{
 "products": [
    {
      "slug": "terraform",
      ...
    },
  ]
}
```

I know this isn't in the context of this PR (and was decided in your spike), but it is something that is making me itch.  If you wanted to file a ticket / defer that's OK.

Reply:
Since adding `CrossProductSection` I'm actually planning to ask about this! When it was decided what was authorable & non-authorable on the page, I believe we decided that the `CrossProductSection` content was totally non-authorable. I can't remember why, but for context that's why in that PR I didn't add a new property under `tutorials-landing.json` for specifying the collection slugs for that section. The JSON file is meant for authorable content and the `constants.tsx` file is meant for non-authorable content. All this to say -- we're on the same page!